tests/unit/containers/edit-profile/professional-info-tab/ProfessionalInfoTab.spec.jsx

```diff
@@ -3,7 +3,7 @@ import { fireEvent, screen } from '@testing-library/react'
 import { renderWithProviders } from '~tests/test-utils'
 import { vi } from 'vitest'
 import { UserRoleEnum } from '~/types'
-import { initialProfessoinalBlock } from '~/redux/features/editProfileSlice.ts'
+import { initialProfessoinalBlock, initialAboutStudent } from '~/redux/features/editProfileSlice.ts'
```

Comment:
Consider fixing the typo in `initialProfessionalBlock`.

Reply:
It's not related to your PR, so it's up to you.

Reply:
hi! what do you mean about the `typo`, is it the naming?